src/nvim/buffer_defs.h

```diff
@@ -1211,6 +1211,8 @@ struct window_S {
   colnr_T w_old_visual_col;         ///< last known start of visual part
   colnr_T w_old_curswant;           ///< last known value of Curswant
 
+  linenr_T w_last_cursor_lnum_rnu;  ///< used to track when cursor moves vertically
```

Comment:
Moving this between `w_last_cursorline` and `w_last_cursormoved` looks better, also the comment can be clearer:
```suggestion
  linenr_T w_last_cursor_lnum_rnu;  ///< cursor lnum when 'rnu' was last redrawn
```